src/Symfony/Component/VarDumper/Dumper/CliDumper.php

```diff
@@ -83,6 +84,8 @@ public function __construct($output = null, string $charset = null, int $flags =
             ]);
         }
 
+        $this->handlesHrefGracefully = 'JetBrains-JediTerm' !== getenv('TERMINAL_EMULATOR');
```

Comment:
why moving this up? if not needed, it should be reverted
I suppose you forgot to keep the check for KONSOLE_VERSION? (see removed line)